specificatie/zoek-personen.yaml

```diff
@@ -0,0 +1,203 @@
+          example: "de"
+        voornamen:
+          description: |
+            De verzameling namen die, gescheiden door spaties, aan de geslachtsnaam voorafgaat. ** Bij deze query-parameter is het gebruik van een [wildcard](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature) toegestaan in combinatie met minimaal 2 karakters.** **Zoeken met tekstvelden is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/case_insensitive.feature).**
+          type: string
+          pattern: ^(\?+|\*)?[a-zA-Z0-9À-ÿ' '\-\']{1,200}(\?+|\*)?$
+          example: "Dirk"
+        gemeenteVanInschrijving:
+          description: |
+            Een code die aangeeft in welke gemeente de persoon woont, of de laatste gemeente waar de persoon heeft gewoond, of de gemeente waar de persoon voor het eerst is ingeschreven.
+          type: string
+          pattern: ^[0-9]{4}$
+          example: "0518"
+        huisletter:
+          description: |
+            Een toevoeging aan een huisnummer in de vorm van een letter die door de gemeente aan een adresseerbaar object is gegeven.
+          type: string
+          pattern: ^[a-zA-Z]{1}$
+          example: "a"
+        huisnummer:
+          description: |
+            Een nummer dat door de gemeente aan een adresseerbaar object is gegeven.
+          type: integer
+          minimum: 1
+          maximum: 99999
+          example: 14
+        huisnummertoevoeging:
+          description: |
+            Een toevoeging aan een huisnummer of een combinatie van huisnummer en huisletter die door de gemeente aan een adresseerbaar object is gegeven.
+          type: string
+          pattern: ^[a-zA-Z0-9]{1,4}$
+          example: "bis"
+        nummeraanduidingIdentificatie:
+          description: |
+            Unieke identificatie van een nummeraanduiding (en het bijbehorende adres) in de BAG.
+          type: string
+          pattern: ^[0-9]{16}$
+          example: "0518200000366054"
+        straat:
+          description: |
+            Een naam die door de gemeente aan een openbare ruimte is gegeven. **Gebruik van de wildcard is toegestaan. Zie [feature-beschrijving](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature)** **Zoeken met tekstvelden is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/case_insensitive.feature).
+          type: string
+          pattern: ^(\?+|\*)?[a-zA-Z0-9À-ÿ '\-\']{1,80}(\?+|\*)?$
+          example: "Tulpstraat"
+        postcode:
+          description: |
+            De door PostNL vastgestelde code die bij een bepaalde combinatie van een straatnaam en een huisnummer hoort.
+          type: string
+          pattern: ^[1-9]{1}[0-9]{3}[ ]?[A-Za-z]{2}$
+          example: "2341SX"
+    IngeschrevenPersoonBeperktHalCollectie:
+      type: object
+      properties:
+        _links:
```

Comment:
deze bevat alleen een self-link. die gaat altijd gelijk zijn aan "/personen", omdat alle parameters in de body zitten. heeft dat wel zin?

Reply:
Ik dacht dat _links volgens de JSON HAL spec een required property moest zijn. Dat is niet het geval. Wordt verwijderd